<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newConnectionDlg</class>
 <widget class="QDialog" name="newConnectionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Connection</string>
  </property>
  <widget class="QLabel" name="newDbLbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>118</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Database Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbNameEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>115</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dbLocationLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>181</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Database Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pathBtn">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>179</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbPathEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clBtn">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
